website/sidebars.json

```diff
@@ -6,91 +6,74 @@
       "label": "Quickstart guides",
       "items": ["quickstart_desktop", "quickstart_web", "quickstart_cli"]
     },
-    "user_guide",
-    "config_guide",
     {
       "type": "category",
-      "label": "Config reference",
+      "label": "User guide",
```

Comment:
might want to make this and the config guide plural like the developer guides, or make developer guides singular now that they all feature multiple pages of content

Reply:
made singular